packages/snowpack/src/build/build-import-proxy.ts

```diff
@@ -182,21 +182,19 @@ export async function wrapImportProxy({
   config: SnowpackConfig;
 }) {
   const {baseExt, expandedExt} = getExt(url);
+
   if (baseExt === '.json') {
     return generateJsonImportProxy({code, hmr, isDev, config});
   }
-  if (expandedExt.endsWith('.module.css')) {
-    return await generateCssModuleImportProxy({
-      url,
-      code,
-      isDev,
-      hmr,
-      config,
-    });
-  }
+
   if (baseExt === '.css') {
-    return generateCssImportProxy({code, hmr, isDev, config});
+    // if proxying a CSS file, remove its source map (the path no longer applies)
+    const sanitized = code.replace(/\/\*#\s*sourceMappingURL=[^/]+\//gm, '');
+    return expandedExt.endsWith('.module.css')
```

Comment:
The more I wrestle with this, the more I start to agree with @FredKSchott where we toss out the idea I tried to introduce of a `baseExt` (`.css`) and `expandedExt` (`.module.css`). It’s hard knowing where to start. I think going forward I’d like to keep it simple, and always assume we want only the final extension. And handle compound  extensions (`.module.css`, `.proxy.js`, `.js.map`) on a case-by-case basis.